README.md

```diff
@@ -93,16 +95,20 @@ Other optional fields are skipped and not included in the Teams message.
 
 #### Swagger UI
 
-Accessing to `<Host>:<Port>` (e.g. `localhost:8089`) in a web browser shows the API documentation.
+Accessing to `<Host>:<Port>` (e.g. `localhost:8001`) in a web browser shows the API v1 documentation.
+
+<img src="assets/swaggerv1.png" alt="Swagger UI" style="width: 600px;"/>
+
+Accessing to `<Host>:<Port>/v2` (e.g. `localhost:8001/v2`) in a web browser shows the API v2 documentation.
 
-<img src="assets/swagger.png" alt="Swagger UI" style="width: 600px;"/>
+<img src="assets/swaggerv2.png" alt="Swagger UI" style="width: 600px;"/>
 
 ## Testing
 
 To run the test suite you should type the following:
 
 ```bash
-# After cloning prom2teams :)
+// After cloning prom2teams :)
```

Comment:
AFAIK # is used to write inline comments in bash :) (https://linux.die.net/Bash-Beginners-Guide/sect_02_02.html)

Reply:
updated